src/Illuminate/Translation/Translator.php

```diff
@@ -163,7 +163,7 @@ public function getFromJson($key, array $replace = [], $locale = null)
 
         // we check to see if the $key has a dot, to be sure that $fallback is not the entire
         // array within a translation file, in case of $key value is the same as file name
-        if (! isset($line) && mb_strpos($key, '.')) {
+        if (is_null($line) && mb_strpos($key, '.')) {
```

Comment:
because the `isset` function returns false for both unset variables and null values and return true for other falsy values.
It can be confusing.